_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,445 @@
+
+### Contribute to the knowledge society
+
+Sharing the GC's source code as information that others can learn from or reuse is our civic responsibility.
+It lowers costs for society.
+It increases knowledge.
+
+The Directive on Open Government reflect the value of documents created by public organisations and encourage their reuse.
+Like data and information, source code constitutes a vast, diverse and valuable pool of resources that can benefit the knowledge society.
+Sharing will help maximise the growth potential of the digital economy, as communicated in the Digital Nation Charter's open market:
+
+> Promote competition in digital markets, for all enterprises regardless of size, including in government procurement, encourage and support a dynamic start-up/scale-up culture in the digital and technology sectors, and promote sustainable economic growth through open markets.
+
+This is why the strategy aims to make it possible for ESDC to share software using a process comparable to that for its documents.
+
+Note:
+
+Work done in the open supports greater transparency and accountability, ensuring that citizens can trust the public service in doing due diligence.
+
+It helps contribute to the greater good by sharing knowledge and to ensure durability of investments done with our citizens money.
+
+### Build a world class public service
+
+Open standards and OSS are about co-creation and building public services that connect seamlessly across organisational silos and borders.
+That is why it is a component in most of the actions in the ISA2 programme.
+Building a responsible open Canada is something we can only do ourselves and the best way is to do it together.
+
+A [Study on OSS governance at the European Commission](https://op.europa.eu/en/publication-detail/-/publication/1aa67278-5464-11ea-aece-01aa75ed71a1) shows that their OSS strategy is in line with the best practices from public services and private companies.
+It confirms that the strengthened policy on the use of OSS is the norm nearly everywhere.
+The study's high-level recommendations are:
+
+- emphasise the use and benefits of open source;
+- create a dedicated open source entity that fosters and measures strategy adoption;
+- improve procurement and product-management processes;
+- establish an open culture;
+- collaborate with communities and the OSS ecosystem;
+- manage legal/licence/intellectual property rights issues; and
+- enhance and develop the technical infrastructure and services.
+
+This strategy is also in line with the [European Commission OSS strategy 2020-2023](https://ec.europa.eu/info/sites/default/files/en_ec_open_source_strategy_2020-2023.pdf) while building on the GC Policies, Directives and the Digital Standards.
+
+The rest of the world has realized the value of working in the open by default.
+Canada has a policy in line with the world's best approaches.
+
+By adopting this strategy, ESDC is taking a stand and moves towards a world class public service.
```

Comment:
The is an opportunity here to add pragmatic advantages as well:

1) Cost benefit.  It would be ideal if we could find a few few that point to the potential for cost saving when OSS is used, licensing in particular.  If we could find a percentage (eg 20% as per Tech Arch's OSS Framework) we could cross reference this with ESDC's total license costs.  Would be easy to put scenarios together to demonstrate how further adoption, say 5% in the first 2 years, of OSS would save the department $XXX per year.

2) Resource Availability.  Would be interesting to research whether there is an embarrassment of riches of talent for OSS technologies (eg. PostGres, MySQL, PHP...LAMP).  The GC has a tendency to procure closed system where there is often few resources which creates a long term dependency on private firms.  By having a broad pool of resources to work on OSS Cots and Apps, recruitment would be simplified.  Furthermore, as adoption of OSS grows, there would be other synergies...reuse of talent, simplified architecture/Loosely coupled, etc.